engine/render/renderer/platform/vulkan/Material.cpp

```diff
@@ -337,26 +371,28 @@ void Material::QueuePropertyUpdate(MSArray<VkWriteDescriptorSet, 10>& writeQueue
 int32_t Material::FindPropertyIndex(Hash::StringId id, PropertiesSlot& slot)
 {
     int32_t foundIdx {-1};
-    slot.properties.MForEachI([&](Property& property, size_t i) {
-        if (property.id == id)
+    for (auto it = slot.properties.CreateIterator(); it; ++it)
+    {
+        if ((*it).id == id)
```

Comment:
`if (it->id == id)`?

Reply:
Added the -> operator to the iterators to enable this.